### PR TITLE
Add support for the Ticwatch Pro(catfish)

### PR DESCRIPTION
**Pull request by MagneFire**

### CODE REVIEW

File: README-system-dir.md [hunk 1]
```diff
@@ -21,15 +21,15 @@ It is very likely that you are currently running an up-to-date version of WearOS
 When you follow this document correctly the flow of system build numbers will be: up-to-date version -> M6E69F -> M9E41V.
 
 Below you can find some links where you can find the system files for ME69F and update file for M9E41V:
-* M6E69F full system: https://androidfilehost.com/?fid=745425885120748838 or http://www.rootjunkysdl.com/files/Huawei%20Watch/Firmware/ray-m6e69f-wear-stock.zip
+* M6E69F full system: https://androidfilehost.com/?fid=745425885120748838 or http://www.rootjunkysdl.com/files/Huawei%20Watch/Firmware/catfish-m6e69f-wear-stock.zip
```
Comment (jrtberlin) [inline README-system-dir.md]: This is specific to sturgeon and should be either replaced or removed.

File: README-system-dir.md [hunk 2]
```diff
@@ -21,15 +21,15 @@ It is very likely that you are currently running an up-to-date version of WearOS
...
 * M9E41V OTA file: https://my.hidrive.com/lnk/GrljFu3i
 
 ### 2.1 Install M6E69F
 
 Use the following commands to downgrade the system, make sure that your watch is currently in fastboot mode:
 
 ```sh
-fastboot flash bootloader bootloader-ray-m6e69f.img
+fastboot flash bootloader bootloader-catfish-m6e69f.img
```
Comment (jrtberlin) [inline README-system-dir.md]: The "m6e69f" is probably sturgeon specific.

**Review by jrtberlin — COMMENTED** [review]
Only a minor documentation error. Everything else looks good to merge.
Thanks!

**Comment by MagneFire** [comment]
Thanks for the feedback!

I've removed the README until we have more accurate instructions.
Also added a patch so that the LCD is turned on whenever the screen transitions into the off state :smile:

**Pull request merged**